### ISSUE TITLE
[BUG] Extremely weird bugs. There is really no precise way to describe all this in just the title.

### ISSUE THREAD

**Issue by PPlayer31** [pizzaboxer/bloxstrap#1695]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.

### What problem did you encounter?

I download the bloxstrapper and launch it. It installs successfully. But then I try to open roblox and play a game and this is what happens:
1) It crashes on Upgrading Roblox
2) It UNINSTALLS THE INSTALLER FR
3) It does not allow me to download it again under the same location
4) If I ever try to install it again, it does not install, it stops either on Connecting to Roblox or Applying Roblox Modifications
5) If I restart, I get to repeat the whole process again
**I ALREADY TRIED UNINSTALLING BLOXSTRAP AND ROBLOX AND REINSTALLING**
This happens with or without fastflags.
Thanks in advance

**Comment by PPlayer31** [comment]
I would like to add that normal roblox works just fine.

**Comment by pizzaboxer** [comment]
> It crashes on Upgrading Roblox

Crashes in what way? Does it just close? Does it show an error message?

> It UNINSTALLS THE INSTALLER FR

Can you elaborate please?

> It does not allow me to download it again under the same location

How so? Does it show a message? Can you elaborate please?

> it stops either on Connecting to Roblox or Applying Roblox Modifications

The next time this happens, get Bloxstrap's latest log file from the `%localappdata%\Bloxstrap\Logs` folder and send it here please.

**Comment by PPlayer31** [comment]
Greetings once again and thank you for the quick reply. I shall explain everything as you asked. I apologize for the insufficient explanation I gave.

> It crashes on Upgrading Roblox
When launching, the bootstrapper enters the "Upgrading Roblox" Phase. Then it CLOSES and UNINSTALLS ITSELF AS WELL AS THE INSTALLER. They both get deleted. I can no longer find bloxstrapper anywhere but in the local appdata and I also cannot open it. In addition, if I try to redownload it, it will not allow me to save it in the same folder I downloaded it before. HOWEVER, with a restart, it is as if nothing happened, the installers are all there. Might be a problem on my end. Still though bloxstrap is just not there and I have to install it again, to face the same consequences.

I just formatted my pc and I will check if this was my setup's fault, because it could be likely.
